src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable';
+import { PeopleContex } from '../../peopleContext';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setError(false);
+    setLoading(true);
+
+    getPeople()
+      .then(res => setPeople(res))
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
```

Comment:
You are not handling loading state and error state properly here. It's better to set loading state to false and error state to true in the catch block.